pkgs/tools/security/quill/default.nix

```
@@ -0,0 +1,45 @@
+{ lib, rustPlatform, fetchFromGitHub, openssl, pkg-config, protobuf, which, buildPackages }:
```

For supporting the build on darwin:
```suggestion
{ lib, stdenv, rustPlatform, fetchFromGitHub, openssl, Security, libiconv, pkg-config, protobuf, which, buildPackages }:
```